brainiak/utils/fmrisim.py

```diff
@@ -1595,6 +1603,66 @@ def _generate_noise_temporal_drift(trs,
         phase = (timepoints / (trs - 1) * cycles * 2 * np.pi) + phaseshift
         noise_drift = np.sin(phase)
 
+    elif basis == 'cos_power_drop':
+
+        # Make a vector counting each TR
+        timepoints = np.linspace(0, trs - 1, trs) * tr_duration
+
+        # Specify the other timing information
+        duration = trs * tr_duration
+
+        # How bases do you have?
+        basis_funcs = int(np.floor(2 * duration))
```

Comment:
This comment is related to that about `power_drop` to help me understand the logic. From the line above, I see `duration` is the total duration of the scan in units of a second, probably. So `2*duration` should also be a quantity of time (in units of a second). Then this `basis_funcs` is used below at my next comment.